data-hibernate-jpa/src/test/groovy/io/micronaut/data/hibernate/jakarta_data/entity/EntityTests.java

```diff
@@ -1573,29 +1563,6 @@ public void testLiteralTrue() {
         }
     }
 
-    public Page<Long> oddsFrom21To(Session session, long max, PageRequest pageRequest) {
```

Comment:
This is unused method and that test that was disabled is passing so I enabled it